src/openalea/eartrack/binarisation_folder.py

```diff
@@ -10,8 +10,7 @@
 import openalea.eartrack.binarisation as bin
 
 
-def init(param_folder, input_folder, output_folder):
-    # images_folder = os.path.join(input_folder, 'images')
+def init(param_folder, input_folder, output_folder, log):
```

Comment:
I will set log to False by default:
def init(param_folder, input_folder, output_folder, log=False):

Reply:
OK